pyworklistserver/__main__.py

```diff
@@ -40,49 +40,63 @@ def _configure_logger(filename, pynetdicom_verbosity: int):
 def _get_serverconfig_from_commandline():
     parser = argparse.ArgumentParser(description='Launch dicom worklist server')
     parser.add_argument(
-        '--ip', 
-        default='127.0.0.1', 
+        '--ip',
+        default='127.0.0.1',
         help='Ip address that this server runs under'
     )
 
     parser.add_argument(
-        '--port', 
-        type=int, 
-        default=104, 
+        '--port',
+        type=int,
+        default=104,
         help='port that this server will listen to'
     )
 
     parser.add_argument(
-        '--aetitle', 
-        default='PyWorklistServer', 
+        '--aetitle',
+        default='PyWorklistServer',
         help='DICOM Application Entity title that identifies the server (Max 16 characters)'
     )
 
     parser.add_argument(
-        '--verbose', 
-        action='store_true', 
+        '--verbose',
+        action='store_true',
         help='Extra logging'
     )
 
     parser.add_argument(
-        '--logfile', 
+        '--logfile',
         help='Path to log file where server activity is logged'
     )
 
+    parser.add_argument(
+        '--seedfile'
+        default='seed.txt'
+        help='Path to seed file where last seed is stored for Reproduction'
+    )
+
+    parser.add_argument(
+        '--reproduce'

```

Comment:
Here, you want the args.reproduce to be False if you don't supply the --reproduce argument. So instead, consider the 'store_true' action which will default to False if it is not provided. 

     parser.add_argument('--foo', action='store_true')

True and False values are better suited for boolean variables than int.

See also https://docs.python.org/3/library/argparse.html#action

Reply:
Done